### ISSUE TITLE
What do you use to obtain the sdk?

### ISSUE THREAD

**Issue by ghost** [aevitas/wotsdk#1]
Can you share the script or process you use to decompile wot's python code?

Is it similar to what is done for EVE's stackless python? https://github.com/wibiti/evedec

**Comment by aevitas** [comment]
Yes, it's very similar. I'm using a highly modified version of pyREtic: https://github.com/MyNameIsMeerkat/pyREtic

I would consider releasing it at some point in the future, but as it stands it's far from simple to use (pyREtic is kind of a pain) - it basically boils down to this:
1. Make sure you have downloaded and extracted https://github.com/MyNameIsMeerkat/pyREtic
2. Create a folder in the newly extracted folder named raw_scripts and copy the scripts folder from res and res_bw from WoT so that you have raw_scripts/res and raw_scripts/res_bw
3. run "python REpdb.py"
4. run "auto_remap raw_scripts/res" and select python runtime 2.6.4
5. Once prompted to, re start the script
6. run "set_project auto-map"
7. run "fs_um_decompile raw_scripts/res"
8. Once finished, copy decompiled code from Projects/auto-map/sourcecode
9. Repeat steps 3-8 using res_bw instead of res

I've submitted a pull request or two to pyREtic in the past to address some issues, but those appear to be ignored unfortunately.

**Comment by ghost** [comment]
Thanks that was helpful.
Also do you know if it's straightforward to inject python code into the game process so as to not leave an obvious file trail? In EVE it can be done with PyGILState manipulation to get your own imports to run.

**Comment by aevitas** [comment]
I know it can be done by making a couple engine calls via injected code, not sure how it would be done by using the PyGILState APIs. You can definitely run arbitrary python code in the game's process though.

**Comment by ghost** [comment]
This is one way to do it in EVE, not sure if it will work for WoT: https://github.com/lmacken/pyrasite/blob/develop/pyrasite/win/inject_python.cpp

You said that you can run arbitrary code for WoT, do you know of a way to do it without putting any files into the game directory?
